templates/example_cidr_lookup_content.csv

```diff
@@ -0,0 +1,7 @@
+network,geoname_id
+1.0.0.0/24,2077422
+1.0.1.0/24,1814991
+1.0.2.0/23,1814991
+1.0.4.0/22,2077456
+1.0.8.0/21,1814991
+1.0.16.0/20,1814991
```

Comment:
nit: newline

Reply:
Done!

Reply:
Sorry, don't know how to submit after commit without re-review